imap/mboxname.c

```diff
@@ -1719,6 +1647,17 @@ EXPORTED int mboxname_isscheduledmailbox(const char *name, int mbtype)
     return res;
 }
 
+/*
+ * If (internal) mailbox 'name' is a user's \Scheduled mailbox
+ * returns boolean
```

Comment:
Better to replace `boolean` with `non-zero`?

Reply:
Done (for all functions)